docker-for-mac/release-notes.md

```diff
@@ -18,6 +18,21 @@ about both kinds of releases, and download stable and beta product installers at
 
 ## Stable Release Notes
 
+### Docker for Mac 1.13.1, 2017-02-09 (stable)
+
+**Upgrades**
+
+- [Docker 1.13.1](https://github.com/docker/docker/releases/tag/v1.13.1)
+- [Docker Compose 1.11.1](https://github.com/docker/compose/releases/tag/1.11.1)
+- Linux kernel 4.9.8
+
+**Bug fixes and minor changes**
+
+- Add link to experimental features
+- New 1.13 cancellable operations should now be properly handled by the Docker for desktop
+- Daemon.json should render nicely in the UI
```

Comment:
Maybe `Daemon.json`? Is the capitalization correct? "nicely"? Actually, how about "Rendering improvements" instead of "nicely"?